apps/web/src/pages/index.css

```
@@ -0,0 +1,268 @@
+.homeIconLink:focus-visible {
+  transform: translateY(-1px);
+  background: color-mix(in srgb, var(--surface) 75%, transparent);
+  border-color: color-mix(in srgb, var(--link) 60%, var(--border));
+  outline: none;
+}
+
+.homeIconLink svg {
+  display: block;
+  width: 1rem;
+  height: 1rem;
+  fill: currentColor;
```

Stylelint (17.3.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)